third_party/2/requests/models.pyi

```diff
@@ -77,10 +77,10 @@ class Request(RequestHooksMixin):
     def prepare(self): ...
 
 class PreparedRequest(RequestEncodingMixin, RequestHooksMixin):
-    method = ...  # type: Any
-    url = ...  # type: Any
-    headers = ...  # type: Any
-    body = ...  # type: Any
+    method = ...  # type: str
+    url = ...  # type: str
+    headers = ...  # type: Dict[str, str]
```

Comment:
Technically an (optional) instance of `structures.CaseInsensitiveDict` is used, which is a `MutableMapping` but is *not* a straight `Dict`.